darts/models/torch_forecasting_model.py

```diff
@@ -104,7 +104,7 @@ def __init__(self,
                  lr_scheduler_cls: torch.optim.lr_scheduler._LRScheduler = None,
                  lr_scheduler_kwargs: Optional[Dict] = None,
                  loss_fn: nn.modules.loss._Loss = nn.MSELoss(),
-                 model_name: str = "torch_model_run",  # TODO: uid
+                 model_name: str = "torch_model_run_" + str(os.getpid()),
```

Comment:
We could also maybe add a timestamp, to make it even more unique across several runs happening e.g. few days appart. For instance something like "<date-HH:MM:SS>_torch_model_run_<PID>"

Reply:
Sure, that sounds good!🔝 I added that with the following formatting `"YYYY-mm-dd_HH:MM:SS_torch_model_run_PID"`, e.g., 2021-06-14_09:53:32_torch_model_run_44607

Reply:
Perfect, thanks